tests/framework-tests/core/js/ModelTransformationTests.js

```diff
@@ -48,7 +48,12 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         floaty2: -9876.789,
         hundredInString: "100",
         floatInString: "12.52",
-        floaty2InString: "-9876.789"
+        floatInStringScale1: "12.5",
+        floatInStringScale0: "13",
+        floaty2InString: "-9876.789",
```

Comment:
so just to clarify, the algorithm strips out trailing zeros. So 0.1 + 0.2 should still evaluate to 0.3 even with a scale of 2.